tsconfig.json

```diff
@@ -1,13 +1,8 @@
 {
   "compilerOptions": {
     // Language Options
-    "target": "ES2017",
-    "lib": [
-      "ES2017",
-      "DOM",
-      "DOM.Iterable",
-      "WebWorker"
-    ],
+    "target": "ES2021",
```

Comment:
Why this change?

Reply:
Intl.ListFormat isn't included in previous targets, es2021 is the first to include it. If there are other possibilities, let me know, since I'm not entirely familiar with how these targets work.

Reply:
Even though it might be questionably related to this PR, I do agree the values for `target` and `lib` need to be kept at the latest release or "ESNext".  We don't use TS to transpile and we type check the code we write which is certainly beyond ES 2017.

I did try to set it to either 2022 or next once and got a bunch of valid type errors.  If 2021 passes checks though, we should keep it IMO.